app/views/posts/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>View to show a single post</h1>
\ No newline at end of file
+<article>
+  <section class="main-post">
+    <h1><%= @post.title %> by <%= User.find(@post.author_id).name %></h1>
+    <div class="post-counters">
```

Comment:
Comments counter and likes counter should be on top right side of the post's show page as it can be seen here in [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB533QXSXBSG55KEMMYYNZ7WY6QEC7Q) wireframe.